src/gui/element.rs

```diff
@@ -1715,7 +1723,7 @@ pub fn menu_container<'a>(
     let version_container = Container::new(version_text)
         .center_y()
         .padding(5)
-        .style(style::NormalForegroundContainer(color_palette));
+        .style(style::HoverableForegroundContainer(color_palette));
```

Comment:
Still one that needs to be changed here. Is everything else good to go outside this change? If so, I'll do a final test before merging.